### PR TITLE
feat: Improved Brazilian Portuguese translations

### PR DESCRIPTION
**Pull request by purp0s3**
## Description

- Better choice of words.
- Added date format.
- Added Week Streak translation.
- Added Longest Week Streak translation.

Ref: #236

### Type of change

- [X] Updated documentation (updated the readme, templates, or other repo files)

## How Has This Been Tested?

- [ ] Tested locally with a valid username
- [ ] Tested locally with an invalid username
- [ ] Ran tests with `composer test`
- [ ] Added or updated test cases to test new features

## Checklist:

- [x] I have checked to make sure no other [pull requests](https://github.com/DenverCoder1/github-readme-streak-stats/pulls?q=is%3Apr+sort%3Aupdated-desc+) are open for this issue
- [X] The code is properly formatted and is consistent with the existing code style
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings

### CODE REVIEW

**Comment by DenverCoder1** [comment]
Looks great, thanks for contributing.

Regarding the date format field, it's only needed if the dates do not appear well without it.

i.e. as of right now

**Comment by purp0s3** [comment]
The default format is correct actually, I didn't check it properly. Should I do another pull request or can you merge it without that line?

File: src/translations.php [hunk 1]
```diff
@@ -144,9 +144,12 @@
         "Present" => "Dziś",
     ],
     "pt-br" => [
+        "date_format" => "j/n[/Y]",
```
Comment (DenverCoder1) [inline src/translations.php:147]:
```suggestion
```

**Comment by DenverCoder1** [comment]
You don't have to create a new pull request. You can update it by making changes on your [purp0s3:improved-brazilian-portuguese](https://github.com/purp0s3/github-readme-streak-stats/tree/improved-brazilian-portuguese) branch.

Or by clicking the commit suggestion button like I see you probably just did.

**Review by DenverCoder1 — APPROVED** [review]
Looks great! Thanks for the contribution :tada:

**Pull request merged**